diff --git a/CMakeLists.txt b/CMakeLists.txt
index d8993e7..52f50ea 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,13 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
-
-if(POLICY CMP0025)
-  cmake_policy(SET CMP0025 NEW)
-endif()
-
-project(MONGO_CXX_DRIVER LANGUAGES CXX)
+cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
+project(MONGO_CXX_DRIVER)
 
 if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
   if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "4.8.2")
